### PR TITLE
feat: [ListV2] Proceeds bar polish

### PR DESCRIPTION
**Pull request by cbachmeier**
1. change title string to “You receive”

2. Change enabled BG outline to ‘action’

3. Add drop shadow to container

4. Adjust CTA size to large; change internal padding to 12 px, 12 px, 12 px, 32 px

Screenshots:
Screen Shot 2023-02-13 at 12 26 19
Screen Shot 2023-02-13 at 12 26 30
Screen Shot 2023-02-13 at 12 27 53

### CODE REVIEW

File: src/nft/components/bag/profile/ListingModal.css.ts [hunk 1]
```diff
@@ -29,20 +29,18 @@ export const sectionDivider = style([
```
Comment (cbachmeier) [inline src/nft/components/bag/profile/ListingModal.css.ts:29]: Acknowledging that I'm still using VE here. Once the polish tickets are complete I plan on re-writing this component using styled components

Comment (JackShort) [reply src/nft/components/bag/profile/ListingModal.css.ts:29]: based acknowledgment

File: src/nft/components/bag/profile/ListingButton.tsx [hunk 2]
```diff
@@ -89,7 +94,10 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
   ] = useMemo(() => {
     const noMarketplacesSelected = sellAssets.some((asset: WalletAsset) => asset.marketplaces === undefined)
     const missingExpiration = sellAssets.some((asset) => {
-      return asset.expirationTime != null && asset.expirationTime * 1000 - Date.now() < ms`60 seconds`
+      return (
+        asset.expirationTime != null &&
+        (asset.expirationTime * 1000 - Date.now() < ms`60 seconds` || isNaN(asset.expirationTime))
```
Comment (JackShort) [inline src/nft/components/bag/profile/ListingButton.tsx:99]: nit: I think logically `isNaN(asset.expirationTime)` should be the first statement in the union

**Review by JackShort — APPROVED** [review]
lgtm

**Pull request merged**